### PR TITLE
Revert "Revert "XmlParserOpt""

### PR DESCRIPTION
**Pull request by Alkohol4yk**
Reverts Nelyndir/XDCtoMarkdown#3

### CODE REVIEW

**Review by nsubaru — COMMENTED** [review]
Good

**Review by nsubaru — APPROVED** [review]

**Review by MonsterFich — COMMENTED** [review]
\_('_')_/

**Review by nsubaru — CHANGES_REQUESTED** [review]
Bad formating

File: XMLDocGen/XDCParser.cs [hunk 1]
```diff
@@ -50,75 +50,53 @@ public static List<Models.Member> Parse(XmlNode xmlMembers)
...
-            var name = String.Empty;
             var summary = String.Empty;
-            name = memberName.Substring(2);
-            try
+
+            if (member != null )
             {
-                summary = member.SelectSingleNode("summary").InnerText;
+                var temp = member.SelectSingleNode("summary");
+                    if(temp!= null)
```
Comment (nsubaru) [inline XMLDocGen/XDCParser.cs:63]: Add brackets

**Review by nsubaru — CHANGES_REQUESTED** [review]

**Review by Nelyndir — APPROVED** [review]

**Review by nsubaru — APPROVED** [review]

**Pull request merged**